CHANGELOG.md

```diff
@@ -11,6 +11,27 @@
 
 * Add intervention component ([PR #2196](https://github.com/alphagov/govuk_publishing_components/pull/2196))
 
+* Add scroll tracking to welsh brexit child taxon pages ([PR #2201](https://github.com/alphagov/govuk_publishing_components/pull/2201))
+
+## 24.18.4
```

Comment:
We don't keep merge commits on main branches except for merging of PRs - this can be rebased.